examples/pipelines/simple_pipeline/components/load_from_hub/src/main.py

```diff
@@ -46,9 +44,7 @@ def load(self, args):
         """
         # 1) Load data, read as Dask dataframe
         logger.info("Loading dataset from the hub...")
-        dataset = load_dataset(args.dataset_name, split="train")
-        dataset.to_parquet("data.parquet")
-        dask_df = dd.read_parquet("data.parquet")
+        dask_df = dd.read_parquet(f"hf://datasets/{args.dataset_name}")
```

Comment:
What is the format of the argument we're expecting here now?

Reply:
The name of a dataset on the HF hub, like `lambdalabs/pokemon-blip-captions`.